src/datasources/balances-api/coingecko-api.service.spec.ts

```diff
@@ -168,13 +168,15 @@ describe('CoingeckoAPI', () => {
     });
 
     const expectedCacheDir = new CacheDir(
+      // @ts-expect-error - TODO: remove after migration
```

Comment:
I chose to expect and error over assertion or optional chaining as this will error when we remove the `optional` added here. The others wouldn't.